units/src/amount/unsigned.rs

```diff
@@ -101,7 +101,8 @@ impl Amount {
     ///
     /// The function panics if the argument multiplied by the number of sats
     /// per bitcoin overflows a `u64` type.
```

Comment:
In 047ef24b085f16b16f58fbc041e4f433427d8409:

Should update this comment to say that it panics if the argument exceeds `MAX_MONEY`. Or you can even explicitly say "exceeds 21 million".

Reply:
The new `MAX_MONEY` invariant is not enforced yet.  Currently this comment is correct for the code.

Reply:
Since Amount::MAX and SignedAmount::MAX have been updated, I think it's fair to say it should be done by the next release.

Reply:
Ok, I'm fine accepting this for now.